File: Dayfive/python/card_validator.py
acccount_number = input("Enter account number")
count = 0
if len(acccount_number) != 19:
	print("false")
if acccount_number[4]:
	count+=1
if acccount_number[9]:
	count+=1
if acccount_number[14]:
	count+=1
if len(acccount_number) == 19:
	if acccount_number[0] == '4' or acccount_number[0] == '5' or acccount_number[0] == '6':
		if count == 3:
			print("True")
		else:
			print("false")

File: Dayfive/js/cardvalidator.js
const prompt = require("prompt-sync")();
accountNumber = prompt("Enter account number: ");
if(accountNumber.length != 19)console.log("False");
let counter = 0;
for(let count = 4; count<18;count+=5){
	if(accountNumber.charAt(count) == '-'){
		counter++;
	}
}
if(accountNumber.length == 19){
if(accountNumber.charAt(0) == '4'||accountNumber.charAt(0) == '5'||accountNumber.charAt(0) == '6'){
	if(counter == 3){
		console.log("True");
			}
		else{console.log("False");}
		}
	else{console.log("False");}
		}
else{console.log("False");}	
